discovery/zookeeper/zookeeper.go

```diff
@@ -0,0 +1,228 @@
+package zookeeper
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/joyent/containerpilot/utils"
+	"github.com/samuel/go-zookeeper/zk"
```

Comment:
I think we only moved some of the items I mentioned in https://github.com/joyent/containerpilot/pull/214#discussion_r76603448. Anything that isn't stdlib should be in the 2nd group below.